app/src/main/java/org/mozilla/focus/screenshot/ScreenshotViewerActivity.java

```diff
@@ -221,6 +221,7 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
     }
 
     private void setupView(boolean existed) {
+        mProgressBar.setVisibility(existed ? View.VISIBLE : View.GONE);
```

Comment:
UX requires the progress bar must wait 0.7s to show if image not load completely.
If set the progress bar visibility in setupView(), it will show immediately.